orderer/consensus/hlmirbft/node.go

```diff
@@ -102,32 +114,41 @@ func (n *node) start(fresh, join bool) {
 			n.Node = *node
 		}
 
-		initialNetworkState := InitialNetworkState(n.metadata.ConsenterIds)
+		initialNetworkState := InitialNetworkState(len(n.chain.opts.Consenters))
 		// TODO(harrymknight) Tick interval is fixed. Perhaps introduce TickInterval field in configuration options
-		err = n.ProcessAsNewNode(n.chain.doneC, n.clock.NewTicker(10*time.Millisecond).C(), initialNetworkState, []byte("first"))
-
+		go func() {
+			err := n.ProcessAsNewNode(n.chain.doneC, n.clock.NewTicker(2*time.Second).C(), initialNetworkState, []byte("first"))
+			if err != nil {
+				n.logger.Error(err, "Failed to start mirbft node")
+			}
+		}()
 	} else {
 		n.logger.Info("Restarting mirbft node")
-		n.RestartProcessing(n.chain.doneC, n.clock.NewTicker(10*time.Millisecond).C())
+		go func() {
+			err := n.RestartProcessing(n.chain.doneC, n.clock.NewTicker(2*time.Second).C())
+			if err != nil {
+				n.logger.Error(err, "Failed to restart mirbft node")
+			}
+		}()
 	}
 }
 
-func InitialNetworkState(ConsenterIds []uint64) *msgs.NetworkState {
-	var nodes []uint64
-	for i := 0; i < len(ConsenterIds); i++ {
-		nodes[i] = ConsenterIds[i]
+func InitialNetworkState(nodeCount int) *msgs.NetworkState {
```

Comment:
raise a ticket to put this back to how it was before with a list of consenter ids. Don't do it now.

Reply:
https://fly2plan.atlassian.net/browse/FLY2-153